gradle.properties

```diff
@@ -20,4 +20,7 @@ android.enableJetifier=true
 android.useAndroidX=true
 
 # https://developer.android.com/studio/releases/gradle-plugin#disable-component-creation
-android.disableAutomaticComponentCreation=true
\ No newline at end of file
+android.disableAutomaticComponentCreation=true
+
+# https://github.com/kt3k/coveralls-gradle-plugin/issues/85
+systemProp.jdk.tls.client.protocols="TLSv1,TLSv1.1,TLSv1.2"
```

Comment:
Ehh, it's not very nice, it makes connection less secure, but I think we have no way to fix it without changing code coverage service